roles/ingress/defaults/main.yml

```diff
@@ -32,9 +32,9 @@ ingress_class_name: "{{ atmosphere_ingress_class_name }}"
 #               the ingress role to pass annotations to the
 #               Ingress resource.  You should use role-specific
 #               variable to pass annotations or use the
-#               ingress_global_annotations variable to apply
+#               ingress_default_annotations variable to apply
 #               annotations to all Ingress resources.
 ingress_annotations: {}
 
-# List of annotations to apply to all Ingress resources
-ingress_global_annotations: {}
+# List of annotations to apply to all Ingress resources as default
+ingress_default_annotations: "{{ atmosphere_ingress_annotations }}"
```

Comment:
We should maybe do this:

```
ingress_default_annotations: "{{ ingress_global_annotations | default(atmosphere_ingress_annotations) }}"
```

or maybe it's the other way around?

Reply:
i totally agree with you.